README.md

```diff
@@ -29,7 +36,7 @@
 
 ## Live Demo
 
-[Live Demo Link](https://Okechukwu-muokwugwo/Okechukwu.github.io/)
```

Comment:
- [ ] Would you please provide an actual link to the private loom video presenting the features of the project you built as this is a project documentation requirement and good practice. 
- [ ] Would you please align the items of your footer to match the Behance design as this is a project requirement. The logo should be aligned to the left and the text aligned to the right

### Behance design

![image](https://user-images.githubusercontent.com/98453979/190260226-229fe413-9393-42ff-9462-c73601eae679.png)

### Your design

![image](https://user-images.githubusercontent.com/98453979/190260576-52faa6bd-c3ff-48e3-9a29-77a9b6a0a69d.png)

- [ ] Would you  also ensure that the footer of your about page has a dark background color  to match the Behance design

### Behance design

![image](https://user-images.githubusercontent.com/98453979/190261257-d931bead-cb11-4e9b-83dc-f1c2ba27a99b.png)

### your design

![image](https://user-images.githubusercontent.com/98453979/190261424-8ffd84de-cbda-4b2f-8b17-2f44fb38a333.png)

- [ ] Would you please provide background images to the headline section of your page and adjust the spacing of elements ( on the mobile version) to match the Behance design. You can also add a bold descriptive text  and style it to make it look more appealing 👍 

### Your design

![image](https://user-images.githubusercontent.com/98453979/190264453-b6148ec2-d0a4-401d-9e7a-bb26132f1794.png)

### Behance Design

![image](https://user-images.githubusercontent.com/98453979/190264613-db37ede0-1989-4a9d-96fe-1034518426dd.png)